runtime/vm/jvminit.c

```diff
@@ -5175,6 +5175,50 @@ static void testOptionValueOps(J9JavaVM* vm) {
 	TEST_INT(uResult, (1024*1024*1024));
 	TEST_INT(intResult, OPTION_OK);
 
+#if defined(J9VM_ENV_DATA64)
+	SET_TO(1, "-Xfog8388607T");			/* 2^63 */
```

Comment:
Shouldn't the boundary be at 16777216T (2^24 T), meaning 16777215T should be accepted?
The code uses UDATA which (on 64-bit platforms) holds values up to 2^64-1. After shifting right 40 bits, the maximum value should be 2^24-1.

Reply:
@keithc-ca you were right, `16777215T `(=`2^24-1` indeed) is the maximum size that `vmarg.c` can accept with 'T'/'t' suffix.  My unit tests  on 't'/'T' are now all in "Succeeded" status, thank you very much for your review!